forch/forchestrator.py

```diff
@@ -97,8 +92,16 @@ def initialize(self):
         self._process_static_device_placement()
         self._process_static_device_behavior()
 
+        if not self._behavior_config_file:
+            self._behavior_config_file = os.path.join(
```

Comment:
you are assigning this variable in two places, it should only be one

Reply:
Added self._calculate_behavioral_config() where self._behavior_config_file is assigned